notifications/service/service.go

```diff
@@ -0,0 +1,34 @@
+package service
```

Comment:
after you have moved the main to lambda/xx then you don't need the extra service folder and you can put everything inside /notifications directly